guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,246 @@ functional_areas:
+1. **Modify Magento variables**—specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables in the `magento-vars.php` file
+1. **Deploy**—deploy and test the `integration` branch
 
-### Configure routes
+### Configure routes for separate domains {#routes}
 
-Magento Enterprise Edition *routes* define how incoming URLs are processed. The way you configure routes depends on how you want your site to operate. We suggest configuring routes for integration as follows. You can edit the values later if your needs change.
+Routes define how to process incoming URLs. Multiple stores with unique domains requires you to define each domain in the `routes.yaml` file. The way you configure routes depends on how you want your site to operate.
 
-{:.bs-callout .bs-callout-info}
+{:.bs-callout-info}
 For Pro, you must create a [Support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to set up routes in the Staging or Production environment.
 
 #### To configure routes in an integration environment:
 
-1.  On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
-
-1.  Change to your {{site.data.var.ee}} base directory.
-
-1.  Open the `.magento/routes.yaml` file in a text editor.
-
-1.  Replace the contents with the following:
-
-    ```yaml
-    "http://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "http://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-    ```
-
-1.  Save your changes to the `routes.yaml` file.
+1. On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+
+1. Change to your {{site.data.var.ee}} base directory.
+
+1. Open the `.magento/routes.yaml` file in a text editor.
+
+1. Replace the contents with the following:
+
+   ```yaml
+   "http://{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://{default}/":
+       type: upstream
+       upstream: "mymagento:https"
+
+   "http://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:https"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+### Configure locations for shared domains {#locations}
+
+Where routes defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share that domain.
+
+```yaml
+web:
+    locations:
+        "/": &app
```

Comment:
I do not know how, but we need to mark that we added alias `app` to "/" and use it for `/<second-site>` for avoiding duplication